app/client/src/features/events/EventLive.tsx

```diff
@@ -112,10 +135,93 @@ export function EventLive({ eventLiveQueryRef, validateInviteQueryRef }: EventLi
         });
     };
 
+    // mutation to stop the event (set isActive to false)
+    const END_EVENT_MUTATION = graphql`
+        mutation EventLiveEndEventMutation($eventId: String!) {
+            endEvent(eventId: $eventId) {
+                message
+            }
+        }
+    `;
+    // mutation to start the event (set isActive to true)
+    const [commitEventEndMutation] = useMutation<EventLiveEndEventMutation>(END_EVENT_MUTATION);
+    const START_EVENT_MUTATION = graphql`
+        mutation EventLiveStartEventMutation($eventId: String!) {
+            startEvent(eventId: $eventId) {
+                message
+            }
+        }
+    `;
+    const [commitEventStartMutation] = useMutation<EventLiveStartEventMutation>(START_EVENT_MUTATION);
+    const [buttonText, setButtonText] = React.useState(node.isActive ? 'End' : 'Start');
+
+    const [commit] = useMutation<EventLiveMutation>(BROADCAST_MESSAGE_MUTATION);
+    const [broadcastMessage, setBroadcastMessage] = React.useState('');
+
+    const handleSubmit = (event: { preventDefault: () => void }) => {
+        event.preventDefault();
+        try {
+            commit({
+                variables: { input: { eventId, broadcastMessage } },
+                onCompleted(payload) {
+                    if (payload.createBroadcastMessage.isError) displaySnack('Something went wrong!');
+                    else displaySnack('broadcasted message successfully!');
+                },
+            });
+        } catch (err) {
+            displaySnack(err.message);
+        }
+    };
+
+    React.useEffect(() => {
+        if (!node.isActive && !node.isViewerModerator) router.push('/events/' + eventId + '/pre');
+    }, [eventId, node.isActive, node.isViewerModerator, router]);
```

Comment:
Probably want to move this effect since we no longer want to be refreshing the whole event live query.  A better alternative would be moving this to the `eventDetailsCard` and adding isActive to that fragment (which has a refresh interval thus will stay up to date).

Reply:
done